tests/test_itd_device_group_node.rb

```diff
@@ -0,0 +1,215 @@
+# Copyright (c) 2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/itd_device_group_node'
+
+include Cisco
+# TestInterface - Minitest for general functionality
+# of the ItdDeviceGroup class.
+class TestItdDeviceGroupNode < CiscoTestCase
```

Comment:
Use `@skip_unless_supported = 'itd_device_group'` here so as to automatically skip all tests on an unsupported platform. This will also remove the need for the `n79k_platform?` check throughout this class.